### ISSUE TITLE
Role names not translated to Arabic in profile page in OJS3.1.0.1

### ISSUE THREAD

**Issue by alireza08** [pkp/pkp-lib#3223]
Hi,
I've installed a fresh dual language (French/Arabic) journal using OJS3.1.0.1.
The problem is that role names are not translated correctly in workflow tabs.
If I set French as the primary language, role names will always be shown in French even after I change journal's language and If I set Arabic as the primary language, role names are displayed in English!
The locale file for Arabic, contains the correct keys, like
`<message key="default.groups.name.reader">قارئ</message>`
and all other keys are translated, except role names (attached screenshot)

![untitled](https://user-images.githubusercontent.com/7510737/34348113-9d68cc82-ea1d-11e7-9b49-22b245a32565.png)

**Comment by bozana** [comment]
Hi @alireza08 

I think those settings are installed/inserted into DB table user_group_settings when a journal is installed. If those locale keys did not exist at that time they will be missing. Thus, they are probably missing in your DB table user_group_settings for that journal group IDs, correct? I.e. do they also miss on the Users & Roles > Roles page?

Thus, there should be a way to install/reload them also after the journal installation.
@asmecher, if there are more than one journals in the installation, the journal Settings > Website > Languages does not contain the possibility to reload the defaults and the Administration > Site Settings > Languages does not consider the journals. S. https://github.com/pkp/pkp-lib/blob/master/templates/controllers/tab/settings/languages/languages.tpl. Thus, should maybe the journal Settings > Website > Languages also contain the row actions?

**Comment by alireza08** [comment]
Hi @bozana ,
yes, you were correct. I manually edited `user_group_settings` table and role names are correctly displayed.
Also, in another journal which was upgraded from OJS2.4.8, I found that in `submission_file_settings` table. the name of the file includes role keys which are not translated in the interface. like this:
`publications, ##default.groups.name.manager##, 3-4386.pdf`

is it ok if I use MySql `update` command to replace all these keys? or is there another way?
